doc/user/environments.xml

```diff
@@ -1856,26 +2068,23 @@ SCons/Tool/SubDir1/SubDir2/SomeTool/__init__.py
 ./site_scons/site_tools/SubDir1/SubDir2/SomeTool/__init__.py
       </sconstruct>
 
-      <para>
-      For python2 It's important to note when creating tools within sub-directories,
-      there needs to be a __init__.py file within each directory.
-      This file can just be empty.
-      This is the same constraint used by python when loading modules
-      from within sub-directories (packages).
-      For python3 this appears to be no longer a requirement.
-      </para>
     </section>
 
 	<section>
     <title>Using sys.path within the toolpath</title>
 
       <para>
-      If we want to access tools externally to scons on the sys.path
-      (one example would be tools installed via the pip package manager)
-      One way to do this is to use sys.path with the toolpath.
+      If we want to access tools externally to scons on the
```

Comment:
Should all "scons" string be &SCons; or something like that?

Reply:
It's my intent.  There is, btw, no absolute divide between &SCons; and &scons;.  I've kind of divided those as &SCons; is more conceptual, referring to the project, while &scons; refers more to the command you would invoke (since you invoke the command by lowercase name), but I don't promise there's really consistency in that.

Reply:
Should be done now.